var/spack/repos/builtin/packages/ecoslim/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+class Ecoslim(CMakePackage):
+    """EcoSLIM is a Lagrangian, particle-tracking code that simulates
+    advective and diffusive movement of water parcels.  EcoSLIM
+    integrates seamlessly with ParFlow-CLM."""
+
+    homepage = "https://github.com/reedmaxwell/EcoSLIM"
+    url      = "https://github.com/reedmaxwell/EcoSLIM/archive/refs/tags/v1.3.tar.gz"
+    git      = "git@github.com:reedmaxwell/EcoSLIM.git"
```

Comment:
For consistency with nearly all other Spack packages:

```suggestion
    git      = "https://github.com/reedmaxwell/EcoSLIM.git"
```